src/parent-controller.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.parent', ['openfin.window'])
+        .controller('ParentCtrl', ['$scope', 'windowCreationService', function($scope, windowCreationService) {
+            var config = {
+                'autoShow': true,
+                'minWidth': 918,
+                'minHeight': 510,
+                'defaultWidth': 1280,
+                'defaultHeight': 720,
+                'frame': false,
+                'url': 'index.html'
+            };
+
+            windowCreationService.ready(function() {
+                // TODO: Restore correct window(s)
+                windowCreationService.createWindow(config, function(newWindow) {
+                    newWindow.show();
+                });
+
+                $scope.$on('updateFavourites', function(event, data) {
```

Comment:
Given this window is never shown, using angular doesn't make a lot of sense I think. Perhaps ditching it here and using bindings is the way to go, yes.